src/gpm-manager.c

```diff
@@ -1773,21 +1772,21 @@ gpm_manager_systemd_inhibit (GDBusProxy *proxy) {
                             );
     if (error == NULL && res != NULL) {
         g_variant_get(res, "(h)", &r);
-	    egg_debug ("Inhibiting systemd sleep res = %i", r);
-        fd = g_unix_fd_list_get (fd_list, r, &error); 
+        g_debug ("Inhibiting systemd sleep res = %i", r);
+        fd = g_unix_fd_list_get (fd_list, r, &error);
         if (fd == -1) {
-            egg_debug("Failed to get systemd inhibitor");
+            g_debug("Failed to get systemd inhibitor");
             return r;
         }
-        egg_debug ("System inhibitor fd is %d", fd);
+        g_debug ("System inhibitor fd is %d", fd);
         g_object_unref (fd_list);
         g_variant_unref (res);
     } else if (error != NULL || res == NULL) {
-        egg_error ("Error in dbus - %s", error->message);
+        g_error ("Error in dbus - %s", error->message);
```

Comment:
```
gpm-manager.c:1785:40: warning: Access to field 'message' results in a dereference of a null pointer (loaded from variable 'error')

        g_error ("Error in dbus - %s", error->message);

                                       ^~~~~~~~~~~~~~
```
https://travis-ci.org/mate-desktop/mate-power-manager/jobs/650313264#L2356
Not sure if this warning comes from you, but you touched the code :P

Reply:
PR to fix it. https://github.com/mate-desktop/mate-power-manager/pull/328
I guess 328 needs to merged before this one.